agrirouter-api-java-api/src/main/java/com/dke/data/agrirouter/api/service/onboard/secured/AuthorizationRequestService.java

```diff
@@ -0,0 +1,26 @@
+package com.dke.data.agrirouter.api.service.onboard.secured;
+
+import com.dke.data.agrirouter.api.dto.registrationrequest.secured.AuthorizationResponse;
+import com.dke.data.agrirouter.api.dto.registrationrequest.secured.AuthorizationResponseToken;
+import com.dke.data.agrirouter.api.service.parameters.AuthorizationRequestParameters;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/** Service for the registration request. */
+public interface AuthorizationRequestService {
+
+  AuthorizationResponse callForAuthorizationResponse(
+      AuthorizationRequestParameters authorizationRequestParameters);
+
+  String getAuthorizationRequestURL(AuthorizationRequestParameters authorizationRequestParameters);
+
+  // Analyse Result from different possible sources
```

Comment:
What is the purpose of such a comment? This is no JavaDoc, therefore a bad smell in code. Never used, never seen in any IDE. Move to JavaDoc or remove.

Reply:
Fixed by removing. Should smell much better now :)